src/modules/rewards/components/RewardsActivityTitle.tsx

```diff
@@ -50,6 +50,27 @@ const RewardsActivityTitle: FC<RewardsActivityTitleProps> = ({ activityTitle, is
             {linkedText}
           </Link>
 
+          <Text
+            variant={variant || 'bl-semibold'}
+            color={color || 'text-primary'}
+            as="span"
+          >
+            {innerText}
+          </Text>
+
+          <Link
+            to={secondUrl}
+            isText
+            target="_blank"
+            textProps={{
+              variant: variant || 'bl-semibold',
+              color: 'text-brand-medium',
+              display: 'inline-block',
+            }}
+          >
+            {secondLinkedText}
+          </Link>
```

Comment:
I think these 2 should be conditional. Right?

```suggestion
         {innerText && <Text
            variant={variant || 'bl-semibold'}
            color={color || 'text-primary'}
            as="span"
          >
            {innerText}
          </Text>}

         {secondUrl &&secondLinkedText && <Link
            to={secondUrl}
            isText
            target="_blank"
            textProps={{
              variant: variant || 'bl-semibold',
              color: 'text-brand-medium',
              display: 'inline-block',
            }}
          >
            {secondLinkedText}
          </Link>}
```